src/disk.ts

```diff
@@ -53,13 +56,27 @@ const writeFileListToStream = async function (
       await streamTransformedFile(file.filename, out, metadata[file.filename].transformed);
     }
   }
+  const unpackedSymlinks = links.filter((f) => f.unpack);
+  for (const file of unpackedSymlinks) {
+    // the symlink needs to be recreated outside in .unpacked
+    const filename = path.relative(filesystem.getRootPath(), file.filename);
+    const link = await fs.readlink(file.filename);
+    await fs.symlink(link, path.join(`${dest}.unpacked`, filename)).catch(async (error) => {
+      if (error.code === 'EPERM' && error.syscall === 'symlink') {
+        throw new Error(
```

Comment:
On Windows, using junction to create symlinks should avoid permission issues.

Reply:
Great callout. I read somewhere (can't find the link atm) that junctions are absolute links, as opposed to a "pointer" relative path. If so, we don't want absolute links between folders in the unpacked folder, otherwise I don't think the asar will work on other computers.